qemu/tests/boot_from_device.py

```diff
@@ -107,10 +108,9 @@ def check_boot_result(boot_fail_info, device_name):
             except ValueError:
                 output = vm.serial_console.get_output()
 
-            for i in infos:
-                if not re.search(i, output):
-                    raise error.TestFail("Could not boot from"
-                                         " '%s'" % device_name)
+            if re.search(infos[1], output):
```

Comment:
it was checking both infos[0] and infos[1], why do you only check the latter now?

Reply:
Check infos[1] is just enough.